spec/index.bs

```diff
@@ -4171,10 +4189,10 @@ dictionary GPUVertexAttributeDescriptor {
 
     Return `true`, if and only if, all of the following conditions are satisfied:
 
-        1. |descriptor|.{{GPUVertexBufferLayoutDescriptor/attributes}}.length is less than or equal to 16.
-        1. |descriptor|.{{GPUVertexBufferLayoutDescriptor/arrayStride}} is less then or equal to 2048.
+        1. |descriptor|.{{GPUVertexBufferLayoutDescriptor/attributes}}.length is less than or equal to {{GPULimits/maxVertexAttributes}}.
```

Comment:
this check should be replaced by a combined check over the whole pipeline (as opposed to a single vertex buffer)